Updates p2 feature version

ujhelyiz opened the pull request.

nikhilnanivadekar commented:
@ujhelyiz can you please rebase on to upstream/master. I think you can drop the 2nd commit as I fixed the Jenkins job to handle the change in version automatically: https://ci.eclipse.org/collections/job/new-version/49/console
Commit which handled the update: https://github.com/eclipse/eclipse-collections/commit/b5b1b319c1bc12db4eb3a60b92707525f1ea3ffa
Can you please review if that commit handled everything?

ujhelyiz commented:
I have rebased the commit and removed my re-versioning; https://github.com/eclipse/eclipse-collections/commit/b5b1b319c1bc12db4eb3a60b92707525f1ea3ffa seemed fine.

nikhilnanivadekar commented:
Perfect! thank you @ujhelyiz ! 
LGTM, merging this.
